gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/VariantConfigurator.groovy

```diff
@@ -33,7 +33,7 @@ class VariantConfigurator {
             task.group = AndroidCommandPlugin.TASK_GROUP + " for variant " + variationName;
         }
 
-        task.apkPath = "${-> variant.outputs[0].packageApplication.outputFile}"
+        task.apkPath = "${-> variant.outputs[0].outputFile}"
```

Comment:
For reference, no, I believe it has not changed in the Android Gradle plugin. The gradle-android-command-plugin just always used the wrong API ;-)